src/reviews/index.scss

```diff
@@ -103,3 +103,37 @@ h1.article-review-status__heading {
 .return-button__icon {
   @include buttons.button-icon();
 }
+
+.summary-text {
+  @include typography.body();
+}
+
+.summary-text__highlight {
+  color: settings.$color-emphasis;
+}
+
+h3.summary-heading {
+  @include typography.summary-heading();
+  font-weight: unset;
+}
+
+.summary-badge-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  list-style: none;
+  padding: 0;
+}
+
+.summary-badge-list__item {
+  @include buttons.action-button();
+  @include typography.summary-badge();
+  margin-bottom: 6px;
+  margin-right: 6px;
+  padding: 5px 10px 6px 10px;
+}
+
+.summary-badge-list__item--active {
+  color: white;
+  background-color: settings.$color-badge-active;
```

Comment:
Needs to be in alphabetical order

Reply:
Quick PR done for this [here](https://github.com/elifesciences/enhanced-preprints/pull/173) - would you mind giving it an approval